modules/EvseV2G/iso_server.cpp

```diff
@@ -1147,32 +1147,50 @@ static enum v2g_event handle_iso_charge_parameter_discovery(struct v2g_connectio
     res->EVSEChargeParameter_isUsed = 0;
     res->EVSEProcessing = (iso2_EVSEProcessingType)conn->ctx->evse_v2g_data.evse_processing[PHASE_PARAMETER];
 
+    struct linked_ac_params {
+        float max_current;
+        int64_t voltage;
+        int64_t pmax;
+    };
+
+    linked_ac_params default_params;
```

Comment:
Looking over your code I think you dont need `default_params` at all.

Reply:
I think you can calculate `voltage` and `pmax` here. But only `pmax` should be stored in a variable to access it twice.